### DISCUSSION TITLE
Is is safe to use my redis for bull queue and also as a cache?

### DISCUSSION THREAD

**Discussion by Rodrigo-JM** [OptimalBits/bull#2516] [Q&A]
Hey guys,

I want to reuse my redis as a cache. Right now it's only being use for my queues.
Was wondering if there might be any conflicts or is it completely safe. Btw, my cache keys wouldn't have anything to do with queues and bull...

My initial feeling is that it is safe, but I am not a bull specialist.

**Comment by manast** [comment]
As long as the keys do not collide it is OK, however, you must use the ```noeviction``` max memory policy for your queues to work properly: https://redis.io/docs/reference/eviction/

**Reply by Rodrigo-JM** [reply]
Gotcha. I reckon this is so that my queues don’t get evicted once I reach maxmemory. In my particular case I won’t be storing not nearly as much data. I just need to cache a single route that has been throwing timeouts on me. Thanks for the response!

**Reply by manast** [reply]
You *must* use the ```noeviction``` max memory policy to avoid strange errors and problems.
